lib/wikidata/fetcher.rb

```diff
@@ -23,16 +29,8 @@ def initialize(h)
       else
         raise 'No id'
       end
-      load_lookup_data!
-    end
-
-    LOOKUP_FILE = 'https://raw.githubusercontent.com/everypolitician/wikidata-fetcher/master/lookup.json'.freeze
-    def load_lookup_data!
-      lookup = JSON.load(
-        open(LOOKUP_FILE), nil, symbolize_names: true, create_additions: false
-      )
-      @@skip = lookup[:skip]
-      @@want = lookup[:want]
+      @@skip = self.class.lookup[:skip]
+      @@want = self.class.lookup[:want]
```

Comment:
Do we actually need to set these in `initialize`? If we have this information in class data (properly), wouldn't it be cleaner to just look them up from that? If this is because we're trying to get a quick win, and fixing this up better is more complex than I'm thinking, then it's probably OK to just do this. But it feels a little like an awkward half-way house at the minute.